cvat-ui/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (C) 2020-2022 Intel Corporation
+// Copyright (C) 2022 CVAT.ai Corporation
 //
 // SPDX-License-Identifier: MIT
+import 'redux-thunk/extend-redux';
```

Comment:
Could you please explain this change?

Reply:
Recently I've been having such type error in cvat-ui:
```
Argument of type 'ThunkAction<{}, AnyAction>' is not assignable to parameter of type 'AnyAction'.ts(2345)
```
on each call to `dispatch`. This import solved the error.